contributing-docs/16_contribution_workflow.rst

```diff
@@ -196,7 +196,9 @@ Step 4: Prepare PR
      and place in either `newsfragments <https://github.com/apache/airflow/blob/main/newsfragments>`__ for core newsfragments,
      or `chart/newsfragments <https://github.com/apache/airflow/blob/main/chart/newsfragments>`__ for helm chart newsfragments.
 
-     In general newsfragments must be one line.  For newsfragment type ``significant``, you may include summary and body separated by a blank line, similar to ``git`` commit messages.
+     In general newsfragments must be one line.  For newsfragment type ``significant``,
+     you should follow the template in ``newsfragments/template.significant.rst`` to include summary, body, change type and migrations rules needed.
+     This can also be done by command ``uv tool run towncrier create --dir . --config newsfragments/config.toml --content `cat newsfragments/template.significant.rst```.
```

Comment:
```suggestion
     This can also be done by command ``uv tool run towncrier create --dir . --config newsfragments/config.toml --content "`cat newsfragments/template.significant.rst`"`  "+.significant.rst"`.
```

Reply:
Two fixes/suggestions in one:

* The `cat` without quotes does not  work in `zsh` on linux at least - it fails with "Error: Got unexpected extra arguments (a short and imperative summary of this changes ... " as the content is passed as arguments
* The `+.significant.rst` will avoid asking questions about issue and type and it will create a random-generated newsfragment prefix - which I think is a good idea - we should likely stop using PR# as newsfragment id as this is always chicken-egg problem - you can only add newsfragment after you create PR - with random prefix we avoid that chicken/egg and there is little value I think with having PR # in newsfragment (you can always git-blame to check the PR # it was merged in).

Reply:
Also another small suggestion - separting the command to separate bash section makes it easier to copy as you get the "copy" button:

```bash
You can copy me easily by single click
```

Reply:
Ah, the `cat` part is something I missed 🤦‍♂️

But I'm not sure about the `+.significant.rst` 🤔  If we want to do it, we probably would like to send it to the dev list before doing it, so I'll keep this part as it it for now. I understand the chicken-egg problem, but yep it doesn't bother me as much. For me to list https://github.com/apache/airflow/issues/41641, the current naming actually helps a lot

Reply:
Not very strong on that one, but that's a bummer/friction  a bit - but if it helps you, I am cool (note that there is already one exception though - not random but `aip-72.significant.rst`

Reply:
yep, but I thought that one was for grouping many significant ones. not sure whether I misunderstand it 🤔